### ISSUE TITLE
[bug]: CommandItem is disabled even if data-disabled is set to false.

### ISSUE THREAD

**Issue by bleedingAyush** [shadcn-ui/ui#4349]
### Describe the bug

I was trying to create team selector so I copied the code from your repo [TeamSwitcher.tsx](https://github.com/shadcn-ui/ui/blob/cc70f6ef4389d93b0606027f6271bcc8140b2d9b/apps/www/app/(app)/examples/dashboard/components/team-switcher.tsx#L81)

After doing everything as expected the CommandItem was not working. After inspecting the devtools I found out that it is disabled
```
.data-\[disabled\]\:pointer-events-none[data-disabled] {
    pointer-events: none;
}
```

```
  <CommandGroup>
                <DialogTrigger asChild>
                  <CommandItem
                    onSelect={() => {
                      setOpen(false);
                      setShowNewTeamDialog(true);
                    }}
                  >
                    <CirclePlus className="mr-2 h-5 w-5" />
                    Create Team
                  </CommandItem>
                </DialogTrigger>
              </CommandGroup>
```

### Affected component/components

CommandItem

### How to reproduce

1. Install shadncn/ui and command
2. Copy the code from shadcn/ui repo for the component [TeamSwitcher.tsx](https://github.com/shadcn-ui/ui/blob/cc70f6ef4389d93b0606027f6271bcc8140b2d9b/apps/www/app/(app)/examples/dashboard/components/team-switcher.tsx#L81)
3. After opening the Command the items are not selectable.

### System Info

```bash
Sec-Ch-Ua-Platform:
"Windows"
User-Agent:
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/126.0.0.0 Safari/537.36 Edg/126.0.0.0
```

### Before submitting

- [X] I've made research efforts and searched the documentation
- [X] I've searched for existing issues

**Comment by jqjjian** [comment]
Yes, I have encountered the same problem, which has not been solved yet.

**Comment by yeonsubak** [comment]
I've submitted a PR for fixing this bug: 
* #4297 

Please check the video in the bug replication steps and if you are experiencing the same bug, here are the fixes:
1. Downgrade cmdk to @0.2.1 on your project's `package.json`. **(Simplest)**
2. Modify CommandItem component as below
    ```typescript
    const CommandItem = React.forwardRef<
      React.ElementRef<typeof CommandPrimitive.Item>,
      React.ComponentPropsWithoutRef<typeof CommandPrimitive.Item>
    >(({ className, disabled, ...props }, ref) => (
      <CommandPrimitive.Item
        ref={ref}
        disabled={disabled}
        className={cn(
          "relative flex cursor-pointer select-none items-center rounded-sm px-2 py-1.5 text-sm outline-none aria-selected:bg-accent aria-selected:text-accent-foreground",
          disabled ? "opacity-50" : '',
          className
        )}
        {...props}
      />
    ))
    ```

**Comment by shadcn** [comment]
This issue has been automatically marked as stale due to one year of inactivity. It will be closed in 7 days unless there’s further input. If you believe this issue is still relevant, please leave a comment or provide updated details. Thank you. (This is an automated message)

**Comment by shadcn** [comment]
This issue has been automatically closed due to one year of inactivity. If you’re still experiencing a similar problem or have additional details to share, please open a new issue following our current issue template. Your updated report helps us investigate and address concerns more efficiently. Thank you for your understanding! (This is an automated message)
